README.md

```diff
@@ -44,45 +64,77 @@ a researcher to maintain a version-controlled R Markdown website:
 
 ## Quick start
 
-* Install [R][r] ([instructions][swc-r] from [Software Carpentry][swc])
-
-* (Optional) Install [RStudio][rstudio] (workflowr takes advantages of some
-RStudio features, but RStudio is not required to use workflowr)
+workflowr builds on several software tools including
+[git](http://cran.r-project.org), [pandoc](http://pandoc.org) and
+[knitr](yihui.name/knitr), **but you do not need to use any of these
```

Comment:
"you do not need to use any of these tools to get started with workflowr" could be misinterpreted that these are not software dependencies of workflowr. How about instead "you do not need to have prior experience with these tools to get started with workflowr"?